assets/extensions/login-template.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!--
+
+This template can be modified and used to customize the login page. To replace
+the login page, set master configuration option oauthConfig.templates.login to
+the path of the template file. Do not change lines with curly braces {{ }} below.
+
+oauthConfig:
+  templates:
+    login: templates/login-template.html
+
+-->
+<html>
+  <head>
+    <title>Login</title>
+  </head>
+  <body>
+
+    {{ if .Error }}
+      <div>{{ .Error }}</div>
+    {{ end }}
+
+    <form action="{{ .Action }}" method="POST">
+      <input type="hidden" name="{{ .Names.Then }}" value="{{ .Values.Then }}">
+      <input type="hidden" name="{{ .Names.CSRF }}" value="{{ .Values.CSRF }}">
+
+      <label for="inputUsername">Username</label>
+      <input type="text" id="inputUsername" placeholder="" tabindex="1" autofocus="autofocus" type="text" name="{{ .Names.Username }}" value="{{ .Values.Username }}">
+
+      <label for="inputPassword">Password</label>
+      <input type="password" id="inputPassword" placeholder="" tabindex="2" type="password" name="{{ .Names.Password }}" value="">
+
+      <button type="submit" tabindex="3">Log In</button>
```

Comment:
feel free to add minimal markup to make this look a tiny bit nicer, like a `<div>` around the username and around the password bits to put them each on their own line, and a tiny stylesheet to make the error red, etc